src/ClassicVersion.java

```diff
@@ -0,0 +1,24 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public class ClassicVersion {
+    public static void fizzBuzzClassic() {
+        List<Integer> numbers = new ArrayList<>();
+
+        for (int i = 0; i < 100; i++) {
+            numbers.add(i + 1);
+        }
+
+        for (Integer number : numbers) {
+            if (number % 3 == 0) {
+                System.out.println(number + " Fizz");
+            }
+            if (number % 5 == 0) {
+                System.out.println(number + " Buzz");
+            }
+            if (number % 3 == 0 && number % 5 == 0) {
+                System.out.println(number + " FizzBuzz");
```

Comment:
In your code, if number is divisible by 3 and 5 you code will print all of the ifs. 
number + Fizz number + Buzz number + FizzBuzz. So this part is wrong.